frontend/packages/dev-console/src/components/import/CreateSecretModal.tsx

```diff
@@ -24,7 +24,6 @@ const CreateSecretModal: React.FC<Props> = ({ close, namespace, save, secretType
 
   return (
     <SecretFormWrapper
-      onCancel={close}
```

Comment:
This seems to have more impact than intended. Now clicking cancel in this dev console secret modal will not close the modal. I think we need to retain support for the optional `onCancel` prop in the SecretFormWrapper component since the API is being used here.

Reply:
> This seems to have more impact than intended. Now clicking cancel in this dev console secret modal will not close the modal. I think we need to retain support for the optional `onCancel` prop in the SecretFormWrapper component since the API is being used here.

Hmm, I traced  where `SecretFormWrapper` is used in `CreateSecretModal`, and I couldn't find any instance that is passing in the `onCancel` props. Are you referring to the "Add secret to workload" in dev-console, or something else?

Reply:
> IIUC, the original behavior of the `SecretFormWrapper` component uses the optional `onCancel` callback prop when the cancel button is clicked. Dev console appears to use the `SecretFormWrapper` component in a modal, and uses this `onCancel` prop to close the modal. So, from what I can tell, removing `onCancel` from `SecretFormWrapper` may cause a regression in the dev console. I think it's beyond the scope of this bug to track down the usage of this specific prop and remove it. We should leave it as it was since there does seem to be some risk of regression.

@TheRealJon Updated! PTAL.